ext/nkf/nkf-utf8/nkf.c

```diff
@@ -41,6 +41,19 @@
 #include <assert.h>
 
 
+#ifndef RB_THREAD_LOCAL_SPECIFIER
+#  if __STDC_VERSION__ >= 201112
+#    define RB_THREAD_LOCAL_SPECIFIER _Thread_local
+#  elif defined(__GNUC__)
+     /* note that ICC (linux) and Clang are covered by __GNUC__ */
+#    define RB_THREAD_LOCAL_SPECIFIER __thread
+#  endif
+#endif
+
+#ifndef RB_THREAD_LOCAL_SPECIFIER
+#  define RB_THREAD_LOCAL_SPECIFIER
+#endif
+
```

Comment:
Probably it makes sense to move this logic into a header file to share in both files. Does it makes sense?

Reply:
You should be able to omit this change, because `ext/nkf/nkf-utf8/nkf.c` is included in `ext/nkf/nkf.c` after these macros are defined in that file: https://github.com/ruby/nkf/blob/4cecd2648e95754964ddb68b354918b70b9358be/ext/nkf/nkf.c#L76

Reply:
Right, thank you!